### PR TITLE
[BEAM-640] Update Python SDK README text

### PR DESCRIPTION
**Pull request by hadarhg**
Update the README for Beam Python SDK:
- [x] Remove references to Alpha release
- [x] Change "Dataflow" references to "Beam"
- [x] Update Getting Started instructions 
- [x] Update the rest of the sections and the TOC

### CODE REVIEW

**Comment by hadarhg** [comment]
R: @francesperry

File: sdks/python/README.md [hunk 1]
```diff
@@ -51,198 +42,103 @@ from the Python programming language.
...
+With your virtual environment active, you can follow along this tour by running a `pydoc` server on a local port of your choosing (this example uses port 8888):
 
-    pydoc -p 8888
+```
+pydoc -p 8888
+```
 
-Now open your browser and go to
+Open your browser and go to
 http://localhost:8888/google.cloud.dataflow.html
```
Comment (francesperry) [inline sdks/python/README.md]: Why hasn't this path changed?

File: sdks/python/README.md [hunk 2]
```diff
@@ -467,33 +354,16 @@ SAMPLE_DATA = [('a', 1), ('b', 10), ('a', 2), ('a', 3), ('b', 20)]
...
-stage the resulting tarball and later install it in the workers executing the
-job.
+Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`. If the code is organized in this fashion, you can use the `--setup_file` command line option to create a source distribution out of the project files, stage the resulting tarball, and later install it in the workers executing the job.
 
 ## Contact Us
 
-We welcome all usage-related questions on
-[Stack Overflow](https://stackoverflow.com/questions/tagged/google-cloud-dataflow)
-tagged with `google-cloud-dataflow`.
+We welcome all usage-related questions on [Stack Overflow](https://stackoverflow.com/questions/tagged/google-cloud-dataflow) tagged with `google-cloud-dataflow`.
```
Comment (francesperry) [inline sdks/python/README.md]: Suggest just removing this. (Should be understood to come in via standard beam channels, given this is a branch on a beam repo.)

File: sdks/python/README.md [hunk 3]
```diff
@@ -467,33 +354,16 @@ SAMPLE_DATA = [('a', 1), ('b', 10), ('a', 2), ('a', 3), ('b', 20)]
...
 
-Please use the
-[issue tracker](https://github.com/GoogleCloudPlatform/DataflowPythonSDK/issues)
-on GitHub to report any bugs, comments or questions regarding SDK development.
+Please report any issues on [JIRA](https://issues.apache.org/jira/browse/BEAM/?selectedTab=com.atlassian.jira.jira-projects-plugin:issues-panel).
```
Comment (francesperry) [inline sdks/python/README.md]: Let's link to the py component: https://issues.apache.org/jira/browse/BEAM/component/12328910

File: sdks/python/README.md [hunk 4]
```diff
@@ -467,33 +354,16 @@ SAMPLE_DATA = [('a', 1), ('b', 10), ('a', 2), ('a', 3), ('b', 20)]
...
+Many projects will grow to multiple source code files. It is recommended that you organize your project so that all code involved in running your pipeline can be built as a Python package. This way, the package can easily be installed in the VM workers executing the job.
 
-Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`.
-If the code is organized in this fashion then you can use the `--setup_file`
-command line option to create a source distribution out of the project files,
-stage the resulting tarball and later install it in the workers executing the
-job.
+Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`. If the code is organized in this fashion, you can use the `--setup_file` command line option to create a source distribution out of the project files, stage the resulting tarball, and later install it in the workers executing the job.
 
 ## Contact Us
```
Comment (francesperry) [inline sdks/python/README.md]: Contact is the wrong tone. Perhaps just 'More information'.

And couldn't hurt to have a call to action to come contribute.

File: sdks/python/README.md [hunk 5]
```diff
@@ -467,33 +354,16 @@ SAMPLE_DATA = [('a', 1), ('b', 10), ('a', 2), ('a', 3), ('b', 20)]
...
-built as a Python package so that it can be installed in the VM workers
-executing a job.
+Many projects will grow to multiple source code files. It is recommended that you organize your project so that all code involved in running your pipeline can be built as a Python package. This way, the package can easily be installed in the VM workers executing the job.
 
-Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`.
-If the code is organized in this fashion then you can use the `--setup_file`
-command line option to create a source distribution out of the project files,
-stage the resulting tarball and later install it in the workers executing the
-job.
+Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`. If the code is organized in this fashion, you can use the `--setup_file` command line option to create a source distribution out of the project files, stage the resulting tarball, and later install it in the workers executing the job.
```
Comment (francesperry) [inline sdks/python/README.md]: should be sdks/python/apache_beam/examples...

Comment (hadarhg) [inline sdks/python/README.md]: Done.

**Comment by hadarhg** [comment]
Thanks, Frances. Done. PTAL.

File: sdks/python/README.md [hunk 6]
```diff
@@ -448,11 +339,7 @@ p.run()
 
 ### Combiner Examples
 
-Combiners are used to create a `PCollection` that contains the sums
-(or max or min) of each of the keys in the initial `PCollecion`.
-Such standard Python functions can be used directly as combiner
-functions. In fact, any function "reducing" an iterable to a
-single value can be used.
+Combiner transforms use functions such as sum, min, or max (or any "reducing" function) to combine multiple values of a PCollection into a single value.
```
Comment (aaltay) [inline sdks/python/README.md]: The description for "reducing" function is lost here. (function "reducing" an iterable to a single value can be used).

**Comment by aaltay** [comment]
LGTM. Thank you.

**Pull request closed without merging**